docs/tutorials/iidm/importerPostProcessor.md

```diff
@@ -0,0 +1,343 @@
+</import>
+```
+
+and configure the groovy script's path to use in the [groovy-post-processor](../../configuration/modules/groovy-post-processor.md) module section, also in the configuration file:
+
+### YAML version
+
+```yaml
+groovy-post-processor:
+       script: <POWSYBL_SAMPLES>/groovyScriptPostProcessor/increase-active-power-postprocessor.groovy
+```
+
+### XML version
+
+```xml
+<groovy-post-processor>
+	<script><POWSYBL_SAMPLES>/groovyScriptPostProcessor/increase-active-power-postprocessor.groovy</script>
+</groovy-post-processor>
+```
+
+3. Configure the `loadFlow`
+
+The configuration for the loadflow is defined in [powsybl configuration file](../../configuration/modules/index.md).
+
+The loadflow implementation to use is read from the `LoadFlowFactory` tag of the `componentDefaultConfig` section.
+
+Here is an example of a minimal configuration for a mock loadflow (i.e. an implementation that does nothing on the network). If you want to execute a true computation, you should configure a 'real' loadflow implementation (e.g. RTE's [Hades2LF](http://www.rte.itesla-pst.org/), is currently free to use for academic/non commercial purposes).
+
+### YAML version
+
+```yaml
+componentDefaultConfig:
+      LoadFlowFactory: com.powsybl.loadflow.mock.LoadFlowFactoryMock
+```
+
+### XML version
+
+```xml
+<componentDefaultConfig>
+    <LoadFlowFactory>com.powsybl.loadflow.mock.LoadFlowFactoryMock</LoadFlowFactory>
+</componentDefaultConfig>
+```
+
+
+
+# Java script (for the JavaScript post processor)
+
+The 'JavaScript' code can be found [here](https://github.com/powsybl/powsybl-core/blob/docs/docs/samples/javaScriptPostProcessor/increaseActivePowerPostProcessor.js).
+
+1 Write a `JavaScript` code that implements the processor's business logic.
+
+```java
+var debug = true; 
+
+function increaseLoadActivePower( load, percent) {
+	if (load != null) {
+		var p = load.getTerminal().getP();
+		load.getTerminal().setP(p * percent);
+		if (debug)
+			print("Load id: "+load.getId() +" Increase load active power, from " + p + " to " +  load.getTerminal().getP());
+	}
+        
+}
+
+var percent = 1.01;
+
+if (network == null) {
+    throw new NullPointerException()
+}
+
+for each (load in network.getLoads()) {
+    increaseLoadActivePower(load , percent); 
+}
+
+```
+
+
+This script uses the network variable, that is binded by the [javaScript](../../iidm/importer/post-processor/JavaScriptPostProcessor.md) post processor.
+
+2 Declare the javaScript post processor (for more details refer to [import](../../configuration/modules/index.md)) in the configuration file:
+
+
+### YAML version
+```yaml
+import:
+    postProcessors: javaScript
+```
+
+
+### XML version
+```xml
+<import>
+    <postProcessors>javaScript</postProcessors>
+</import>
+```
+
+and configure the javaScript code's path to use in the [javascript-post-processor](../../configuration/modules/javaScriptPostProcessor.md) module section, also in the configuration file:
+
```

Comment:
add the yaml version
Remark: could you please correct the page ```../../configuration/modules/javascriptpostprocessor.md``` as well ? I just noticed the configuration was wrong on this page...